windows/devices/uwb/include/windows/devices/uwb/UwbSession.hxx

```diff
@@ -85,6 +85,24 @@ private:
     virtual std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
     GetApplicationConfigurationParametersImpl() override;
 
+    /**
+     * @brief Get the Application Configuration Parameters object
+     *
+     * @param requestedTypes
+     * @return std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+     */
+    virtual std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+    GetApplicationConfigurationParametersImpl(std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> requestedTypes) override;
+
+    /**
+     * @brief Set the Application Configuration Parameters object
+     *
+     * @param params
+     * @return void
```

Comment:
I don't think the `@return void` line is necessary, based on other existing function comments in this file

Reply:
looks like this was added by the docgen. i'll remove